src/MunicipalityRegistry/Municipality/Events/MunicipalityNameWasCleared.cs

```diff
@@ -6,11 +6,16 @@ namespace MunicipalityRegistry.Municipality.Events
     using Newtonsoft.Json;
 
     [EventName("MunicipalityNameWasCleared")]
-    [EventDescription("De naam van de gemeente werd verwijderd voor een bepaalde taal.")]
+    [EventDescription("De naam van de gemeente (in een bepaalde taal) werd gewist.")]
     public class MunicipalityNameWasCleared : IHasProvenance, ISetProvenance
     {
+        [EventPropertyDescription("Interne GUID van de gemeente.")]
         public Guid MunicipalityId { get; }
+        
+        [EventPropertyDescription("Taal (voluit, EN) waarin de officiële naam staat.")]
```

Comment:
`Waarvoor de officiele naam werd gewist`?

Reply:
Corrected in new commit.